main.go

```diff
@@ -264,6 +264,26 @@ The value 'none' can be used to disable access logs.`),
 			Usage: "Dial backend request timeout in seconds",
 		},
 		cli.IntFlag{
+			Name:  "client-read-timeout",
+			Value: 30, // TODO: TBC
+			Usage: "Maximum duration for reading the entire request, including the body",
+		},
+		cli.IntFlag{
+			Name:  "client-read-header-timeout",
+			Value: 30, // TODO: TBC
```

Comment:
@cezarsa @andrewsmedina I left a few `TODO: TBC` so we can discuss the default values for the new timeouts.